### ISSUE TITLE
How to finetune unidiffuser

### ISSUE THREAD

**Issue by dawei03896** [thu-ml/unidiffuser#5]
thanks for your great works! by the way can unidiffuser be finetuned like dreambooth?

**Comment by baofff** [comment]
Yes. Many finetuing algorithms on transformers are already available, such as [https://arxiv.org/abs/2110.04366](https://arxiv.org/abs/2110.04366)
